manticore/ethereum.py

```diff
@@ -342,6 +342,22 @@ def check_bitsize(value, size):
         result.append(ABI.make_function_arguments(*args))
         return reduce(lambda x,y: x+y, result)
 
+    @staticmethod
+    def get_uint(data, offset, size):
```

Comment:
what is this function supposed to do? if it reads a uint out of the data @ offset, why is there a size param?

Reply:
oh, i see now, `uint` in the function name doesn't mean ethereum 256 uint

Reply:
So. uint of different sizes. 
if 'address' -> uint160 -> size:=20.
We need to document it.

Reply:
i'm working on documentation/cleanup PR to help my understanding :)

Reply:
should we change the `size` parameter to be in bytes rather than bits? `size` should always be a multiple of 8, and if it is not, its an error.

Reply:
Oh, size may be bits.

Reply:
when would you want to read a size that is not divisible by 8?